### ISSUE TITLE
[$10][MSFT-32] [Challenge Feed] User can't focus on the 'Community' menu using tab

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2754]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login using valid credentials
2. Press tab key till the 'Community' menu is selected

**Expected Result**
User should be able to select the Community tab using tab

**Actual Result**
User can't focus on the 'Community' menu using tab

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F15

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096077 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096077 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096077 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/topcoder-platform/community-app/pull/2943

**Comment by PrakashDurlabhji** [comment]
@codeMinter request you to review my PR of this issue first because it covers everything and also my commit was before @gets0ul commit for the issue https://github.com/topcoder-platform/community-app/issues/2818 and PR which getsoul raised was https://github.com/topcoder-platform/community-app/pull/2955

ideally issue which getsoul took shall be given to me as approached he used is same as mine the only difference in commits came as i forgot to update testcase and getsoul did at that moment itself.

1. kindly go to getsoul https://github.com/topcoder-platform/community-app/pull/2955 and commit tab see it is 17 hours ago.

2. now go to my PR https://github.com/topcoder-platform/community-app/pull/2943 and commit tab, see it is 22 hours ago.

dont know whether it is conincidence or he took a look at my PR or may be something else, but kindly review my PR first.

thanks

**Comment by tosha5252** [comment]
failed testing at https://test-community-app.topcoder-dev.com/challenges...

Tab skips over 'Community' link....from 'Tracks' to 'SRMs', in the next row...

**Comment by tosha5252** [comment]
Env:

OS: Windows 10
Browser: Chrome

**Comment by lakshmiathreya** [comment]
.. Verified the same on MacOS/Safari - fix not working...

**Comment by PrakashDurlabhji** [comment]
@tosha5252 it is working fine at https://test-community-app.topcoder-dev.com/challenges

it focuses compete tracks and community in sequence.

can you please confirm again?

**Comment by lakshmiathreya** [comment]
Verified focus moves to Community as expected - MacOS/Chrome & Win 10/Edge (browserstack). Moving to QA Pass.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2754 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2754 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2754 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2754 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097539 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097539 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tosha5252** [comment]
Verified working

Env: Windows 10
Browser: Chrome and Edge

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097539<br/><br/>```This is an automated message for crazyk via Topcoder X```
